spec/requests/api/v1/apps_spec.rb

```diff
@@ -9,7 +9,7 @@
     end
 
     let(:client_name)   { 'Test app' }
-    let(:scopes)        { nil }
+    let(:scopes)        { 'read write' }
```

Comment:
Previously we were falling back to default scopes, when the primary way of registering an oauth app is with scopes.